### PR TITLE
dashboard page

### PR DESCRIPTION
**Pull request by mehulsatardekar**
### **Added Dashboard page for Quiz app.**
**Preview** : https://deploy-preview-2--quizgrad.netlify.app/views/home

dashboard

### CODE REVIEW

**Comment by mmanjuprasad** [comment]
The site looks good 
The responsiveness is also good and can be merged

File: public/css/home.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.user-profile{
+    width: 100%;
+    height: 100%;
+    border-radius: 5px;
+    max-height: 11.938rem;
+    object-fit: cover;
+}
+.qg-gap-5{
```
Comment (j24m) [inline public/css/home.css:27]: This class name is not understandable.

File: public/css/home.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+
+.img-text{
+    position: absolute;
+    bottom: 8px;
+    color: var(--img-text);
+    font-weight: 800;
+    padding-left: 10px;
+}
+.qg-flex-1{
```
Comment (j24m) [inline public/css/home.css:48]: this too.

**Review by j24m — CHANGES_REQUESTED** [review]

**Review by Krushna-Kulkarni — COMMENTED** [review]
Look very good. Just a suggestion, user prettier or beautify for code styling.

File: public/css/home.css (hunk 1 above)
Comment (mehulsatardekar) [reply public/css/home.css:27]: Well, this is the **prefix**, for not to collide the actual CSS properties with the component library's  CSS properties, Here basically. **qg** is the initial letter of Quiz and grad

**Comment by mehulsatardekar** [comment]
> Look very good. Just a suggestion, user prettier or beautify for code styling.

Already using prettier, probably instead of making it prettier it is making even worse

**Review by j24m — APPROVED** [review]

**Review by rahulk31 — APPROVED** [review]
Great work Mehul, This is Merge Ready 🚀

File: public/css/home.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    min-height: 12.5rem;
+}
+
+
+.img-text{
+    position: absolute;
+    bottom: 8px;
+    color: var(--img-text);
+    font-weight: 800;
+    padding-left: 10px;
```
Comment (varun1233) [inline public/css/home.css:46]: use rems

File: public/css/home.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+.qg-flex-5{
+    flex: 5;
+}
+.qg-flex-2{
+    flex: 2;
+}
+.category-grid{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr) ) ;
+    gap:10px;
```
Comment (varun1233) [inline public/css/home.css:60]: use rems

**Review by varun1233 — CHANGES_REQUESTED** [review]

**Review by varun1233 — APPROVED** [review]

**Pull request merged**